pages.yml

```diff
@@ -4,3 +4,5 @@
 # Add redirects that have a change with
 # - from: oldname
 #   to: newname
+- from: state-faq
```

Comment:
This config takes the place of https://github.com/18F/federalist-redirects/blob/master/nginx.conf#L84